src/components/atoms/Label/Label.test.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { render, cleanup } from '@testing-library/react';
+import Label from './index';
+
+describe('Test Label Component', () => {
+  afterEach(cleanup);
+  it('renders without crashing', () => {
+    const div = document.createElement('div');
+    ReactDOM.render(<Label />, div);
+  });
+
+  it('renders correct content', () => {
+    const { getByTestId } = render(<Label>Test Content</Label>);
+    expect(getByTestId('labelTest')).toHaveTextContent(/^Test Content$/);
```

Comment:
What about using a `getByLabelText` query and then assert `toBeInTheDocument`?

Reply:
Yes, that's better, thanks!